delivery/app.py

```diff
@@ -6,6 +6,44 @@
 from delivery.error_handling import CustomError, WrongIdError
 from flasgger import Swagger
 from marshmallow import Schema, fields, ValidationError
+from abc import ABC, abstractmethod
+
+
+class AbstractRestaurantRepository(ABC):
+    @abstractmethod
+    def save(self, content, restaurant_id):
```

Comment:
create a `dataclass` named `Restaurant` to work with restaurant entities instead of plain `dict`. place this class to `entities.py` (or `models.py`)

Reply:
fixed